guides/anchor/3-federation-server.md

```diff
@@ -16,7 +16,7 @@ Stellar.org provides a [prebuilt federation server](https://github.com/stellar/g
 
 ## Create a Database
 
-The Stellar federation server is designed to connect to any existing SQL database you might have with a list of account names. It essentially translates a federation request into a SQL query. The server supports MySQL, PostgreSQL, and SQLite3.
+The Stellar federation server is designed to connect to any existing SQL database you might have with a list of account names. It essentially translates a federation request into a SQL query. The server supports PostgreSQL, and SQLite3.
```

Comment:
no need for the `,` between `PostreSQL` and `and`

Reply:
3a2d068